src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class Node<T> {
```

Comment:
The `Node` class should be an inner class within `MyLinkedList`. This will enhance encapsulation and maintainability by keeping the `Node` class closely tied to the `MyLinkedList` implementation.